src/Game.cpp

```diff
@@ -39,17 +39,179 @@ static void sub_53A890(const char *args) {
     sub_612A80(loc_6C3690);
 }
 
+// mc2: 0x005EDA50
+void game_set_window_title(char* title) {
+    HWND hWnd = global_hWnd;
+    global_WindowText = title;
+    if (hWnd != NULL)
+        SetWindowTextA(global_hWnd, title);
+}
+
+// mc2: 0x00539FE0 and 0x00539D90
+void check_sku_version() {
+    global_SKUVersion = 0; //SKU VERSION SLUS-20209
+
+    char* skuVersion = NULL;
+
+    switch (global_SKUVersion) {
+    case 0:
+        skuVersion = "SLUS-20209";
+        break;
+    case 1:
+        skuVersion = "SLES-51054";
+        break;
+    case 2:
+        skuVersion = "SLPM-20209";
+        break;
+    default:
+        // SKU unrecognised
+        mc2_log_error("Unhandled sku");
+        return;
+    }
+
+    mc2_log_C(skuVersion);
+}
+
+void sub_53B9B0() {
+    int esi = 0;
+    if (sub_612E10("pc_480")) {
+        esi = 7;
+    }
+    else if (sub_612E10("pc_720")) {
+        esi = 8;
+    }
+    glo_66315C = esi;
+}
+
+//bool sub_627145(char* path, char* path2) {
+//	//if (glo_86D8A8 == 0) {
+//	//	//627CD0
+//	//	return false;
+//	//}
+//
+//
+//}
+
+int sub_5E1B50(const char* path) {
+    for (int i = 0; i < 10; ++i) {
+        char* language = global_LanguageList[i];
+
+        if (sub_627145(path, language)) {
+            return i;
+        }
+    }
+    return -1;
+}
+
+bool sub_539DE0(const char* path) {
+    for (int i = 0; i < 10; ++i) {
+        char* unk = glo_663160[i];
+
+        if (sub_627145(path, unk)) {
+            glo_6C2C60 = i;
+            return true;
+        }
+    }
+    mc2_log_level_v(LOG_LEVEL_FATAL_ERROR, "mcConfig::SetMode() failed", NULL);
```

Comment:
Yeah it's slightly different in that 6186b0 does a debug break then exits the program. I saw no reason in doing that since this will work in much the same way